src/server/rest/CentralRestServerService.ts

```diff
@@ -96,6 +97,11 @@ class RequestMapper {
           [ServerAction.OCPI_ENPOINT_SEND_EVSE_STATUSES]: OCPIEndpointService.handleSendEVSEStatusesOcpiEndpoint.bind(this),
           [ServerAction.OCPI_ENPOINT_SEND_TOKENS]: OCPIEndpointService.handleSendTokensOcpiEndpoint.bind(this),
           [ServerAction.OCPI_ENPOINT_GENERATE_LOCAL_TOKEN]: OCPIEndpointService.handleGenerateLocalTokenOcpiEndpoint.bind(this),
+          [ServerAction.OICP_ENPOINT_CREATE]: OICPEndpointService.handleCreateOicpEndpoint.bind(this),
```

Comment:
If you already have the related dashboard code, you also have to create a PR on ev-dashboard repo.

Reply:
Okay, at this point I have no code to push.
I can only trigger from the frontend.

Reply:
So how to you test the REST API endpoint code you've implemented?

Reply:
Could you please think about using postman to permit other to use them?

Reply:
I worked on the dashboard and committed my changes to a new branch, also called hubject-implementation
I created a new Roaming Hubject component with a dedicated Tab in the setting.
You can create an endpoint with the base URL https://service.hubject.com
You need the private key and certificate to test the hubject endpoints. (I will send you them)